LA4Falcon.c

```diff
@@ -465,6 +471,15 @@ int main(int argc, char *argv[])
 
     //  For each record do
 
+
+    // debugging
+    setlocale(LC_NUMERIC, "");
+    fprintf( stderr, "about to go into loop with novl = %lld %'d\n", novl, (int) novl );
```

Comment:
I was just testing your code, when I noticed that you have `%'d` instead of `%d`. What is the single-quote for? I see it on other lines too. Just a typo? I've never seen that notation in printf.

Reply:
Oh, I see. That's to apply a field separator, controlled by LC_NUMERIC. Interesting.

* http://wiki.bash-hackers.org/commands/builtin/printf